src/html.rs

```diff
@@ -315,11 +316,11 @@ where
 }
 
 /// This handle keeps the reference to a detached scope to prevent memory leaks.
```

Comment:
docs still call it `handle`.

Am I to understand that this helps with dropping memory when it is no longer used -- because javascript doesn't handle that automatically?

Adding a sentance explaining that would be _really_ useful.